include/libsemigroups/paths.hpp

```diff
@@ -875,6 +522,7 @@ namespace libsemigroups {
   //! Returns the number of paths between a pair of nodes with length in a
   //! given range.
   //!
+  //! \param wg the WordGraph
```

Comment:
```suggestion
  //! \param wg the WordGraph.
```
(and the same for the params below)